tests/base_d_asa/test_open_account.py

```diff
@@ -66,21 +65,10 @@ def test_fail_unauthorized_caller(
     payment = algorand.account.random()
 
     with pytest.raises(LogicError, match=err.UNAUTHORIZED):
-        base_d_asa_client_empty.open_account(
-            holding_address=holding.address,
-            payment_address=payment.address,
-            transaction_parameters=OnCompleteCallParameters(
-                signer=oscar.signer,
-                boxes=[
-                    (
-                        base_d_asa_client_empty.app_id,
-                        DAsaAccountManager.box_id_from_address(oscar.address),
-                    ),
-                    (
-                        base_d_asa_client_empty.app_id,
-                        DAsaAccount.box_id_from_address(holding.address),
-                    ),
-                ],
+        base_d_asa_client_empty.send.open_account(
+            args=(holding.address, payment.address),
```

Comment:
You used different approaches to pass method arguments, sometimes as `tuples` and sometimes with client-generated Args `dataclass`. I'd just use the `dataclass` (omitting `args=`) for consistency.